src/components/Person/Person.jsx

```diff
@@ -1 +1,27 @@
-// export const Person = ({ person }) => ();
+import React from 'react';
+
+export const Person = ({ person }) => {
+  const {
+    name,
+    age,
+    sex,
+    isMarried,
+    partnerName,
+  } = person;
+
+  return (
+    <div className="App">
+      <section className="Person">
+        <h2 className="Person__name">{`My name is ${name}`}</h2>
+        {age
+          ? <p className="Person__age">{`I am ${age}`}</p>
+          : null
+        }
+        {isMarried === true
+          ? <p className="Person__partner">{`${partnerName} is my ${sex === 'm' ? `wife` : `husband`}`}</p>
```

Comment:
CL p2 - Readabily is everything. Format ternary operator operands correctly - move each operand to the separate line

Reply:
I've corrected through variable